docs/chunked_caching.md

```diff
@@ -0,0 +1,43 @@
+# Chunked Caching
+
+## Overview
+
+In some use cases, it may be desirable to reduce the amount of data being transmitted from the cache for small requests. Trickster accomplishes this by *chunking* cache data, or splitting it into subdivisions of a configurable maximum size. Chunking can be configured per-cache and applies to both timeseries and byterange data.
+
+## Configuration
+
+Chunked caching can be enabled and disabled using `use_cache_chunking`:
+
+```yaml
+fs1:
+    cache_type: filesystem
+    provider: filesystem
+    use_cache_chunking: true
+    timeseries_chunk_factor: 420
+    byterange_chunk_size: 4096
+```
+
+`timeseries_chunk_factor` determines the maximum extent of timerange chunks, and `byterange_chunk_size` determines the maximum size of byterange chunks. See [Detail](#detail) for more information.
+
+## Detail
+
+### Timeseries
+
```

Comment:
Add example yaml for chunking in this markdown file as well as in the main example.yaml file